src/simulation.hpp

```diff
@@ -2356,7 +2362,7 @@ template <typename problem_t> auto AMRSimulation<problem_t>::PlotFileMFAtLevel(c
 	}
 	const int ncomp_deriv = derivedNames_.size();
 	const int ncomp_plotMF = ncomp_cc + ncomp_tot_fc + ncomp_deriv;
-	amrex::MultiFab plotMF(grids[lev], dmap[lev], ncomp_plotMF, included_ghosts);
+	amrex::MultiFab plotMF_cc(grids[lev], dmap[lev], ncomp_plotMF, included_ghosts);
```

Comment:
The names of variables inside this function do not need to be changed. It is clear from the name of the function itself that everything is cell centered.

In general, it is strongly preferred not to change existing code unless there are functional changes. Code style and formatting of existing code shouldn't be changed.

Reply:
The reason for this general rule is that unrelated stylistic changes can add significant extra work to the authors of other PRs who have to deal with merge conflicts that could occur after this PR is merged.